instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -214,6 +225,10 @@ def _instrumented_open_call(
                         SpanAttributes.HTTP_FLAVOR
                     ] = f"{ver_[:1]}.{ver_[:-1]}"
 
+                _record_histograms(
+                    histograms, labels, request, result, elapsed_time
+                )
```

Comment:
Will there be a case where the result is None, but we should record the timing regardless of its return value?

Reply:
You are right this line should be outside the condition

Reply:
I changed it so if the result is None we add records only for http_client_duration and http_client_request size,
the status code and HTTP flavor are missing from attributes because we take them from the response.
how does it sound?